robocopkaka/tafrica-copy
# add extra validations for datetime values

#### What does this PR do?
Adds extra validations
#### Description of Task to be completed?
- add validations for start time
- add validations for end time
- add action to toggle active status
- modify seeds file
#### How should this be manually tested?
* You should see more errors when invalid dates and times are passed when creating or editing an event
* You should see a button to toggle the active status on an event's page
#### Any background context you want to provide?
N/A
